### DISCUSSION TITLE
Missing content packages for save not showed, when language is set to chinese

### DISCUSSION THREAD

**Discussion by AzureNocte** [FakeFishGames/Barotrauma#13902] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When language is set to chinese, It will not show what mods are missiong for save, while other languages seem have no such issue.
![屏幕截图(144)](https://github.com/FakeFishGames/Barotrauma/assets/44329789/8ce279b2-87d9-4861-8fa6-bd293e74e0df)
![屏幕截图(143)](https://github.com/FakeFishGames/Barotrauma/assets/44329789/bde7fa81-1548-473f-a229-7beaa0653f55)
![20240429212456](https://github.com/FakeFishGames/Barotrauma/assets/44329789/56e44e3a-0c34-4c2e-974e-427cadeb60b9)

This issue has been since early than 0.21.6, I think, but seems not one mentioned it.

### Reproduction steps

1.Pick a save, disable some mods from its modlist
2.Host server
3.Check information of modlist change for the save

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by 3e849f2e5c** [comment]
Hello thanks for the report! I recently ran a pass through all the language files for any inconsistencies and found many cases, this included.

If you find any more feel free to mention them in this discussion or creating a new discussion for it. We'll be talking with the localization team and hopefully getting them fixed for the next update.

**Reply by 3e849f2e5c** [reply]
I've also taken a note of the other report you posted  https://github.com/FakeFishGames/Barotrauma/discussions/13911

**Reply by AzureNocte** [reply]
Thanks!!!

**Comment by Regalis11** [comment]
Will be fixed in the next update!
